include/dlpack/dlpack.h

```diff
@@ -157,6 +157,20 @@ typedef enum {
   kDLComplex = 5U,
   /*! \brief boolean */
   kDLBool = 6U,
+  /*! \brief FP8 data types. Must set DLDataType.bits=8. */
+  kDLFloat8_e3m4 = 7U,
+  kDLFloat8_e4m3 = 8U,
+  kDLFloat8_e4m3b11fnuz = 9U,
```

Comment:
useful to elaborate a bit on description of each type, and comon usage, e.g. e8m0 can be used in scenarios like microscaling factor in gemm

Reply:
I don't think it is DLPack's responsibility to educate developers which type to pick.